src/main/java/com/gluonhq/substrate/target/AndroidTargetConfiguration.java

```diff
@@ -360,4 +362,20 @@ private void createDevelopKeystore() throws IOException, InterruptedException {
 
         System.err.println("done creating ks");
     }
+    
+    private String findLatestBuildTool(Path sdkPath) throws IOException {
+        Path buildToolsPath = sdkPath.resolve("build-tools");
```

Comment:
`Objects.requireNonNull(sdkPath)` and `Files.exists(sdkPath)`?

Reply:
Adding a check for `Files.exists(buildToolsPath)` makes `Files.exists(sdkPath)` redundant.